bag/search/views.py

```diff
@@ -252,7 +258,8 @@ def find_default_queries(q_select) -> List[Search]:
 def select_queries(
         query_string: str,
         analyzer: QueryAnalyzer,
-        q_select: AbstractSet[str] = ()) -> List[Search]:
+        q_select: AbstractSet[str] = (),
```

Comment:
Bij deze zou ik obv het type als default geen tuple maar een 'set()' verwachten.
Maar, wellicht is een None als default nog beter ivm. het python gedrag dat mutable defaults van parameters problemen kunnen geven (https://docs.python-guide.org/writing/gotchas/)

Reply:
Was geen onderdeel van de wijziging, maar is inderdaad wel beter. Ik zal het veranderen